3scale/porta
# 🐛 Fix double slashes in Integration Show

**What this PR does / why we need it**:

At `Service > Integration > Configuration` the URL may present multiple slashes together (such as `//path/path`). This happens when the Backend API path is `/` or when a method has multiple slashes (this is another bug present, see [THREESCALE-4498](https://issues.redhat.com/browse/THREESCALE-4498))
![Screen Shot 2020-02-14 at 17 26 12](https://user-images.githubusercontent.com/11672286/74612215-8a951800-5103-11ea-94dd-9a1177bb9a64.png)


**Which issue(s) this PR fixes** 

[THREESCALE-4499: Double Slash in Mapping rules at APIcast configuration page](https://issues.redhat.com/browse/THREESCALE-4499)

**Verification steps** 

1. Go to Product page and add  Backend with path `/`.
2. At the Configuration page, there should be no double slash in Mapping Rules.
<img width="1014" alt="Screen Shot 2020-02-16 at 21 38 14" src="https://user-images.githubusercontent.com/11672286/74612329-bd8bdb80-5104-11ea-8c83-2b276f9b7ab6.png">
